src/Func.h

```diff
@@ -70,10 +73,15 @@ class Func : public Obj
 		{
 		detail::StmtPtr stmts;
 		int priority;
+		std::set<EventGroupPtr> groups;
+		int disabled_groups = 0;
```

Comment:
We don't really need the count, do we? Then how about just doing `bool disabled = false` here now?

Reply:
And I'd bring back a comment that this is maintained through the event groups mgmt.